GoMeet/src/main/webapp/js/interactive-map.js

```diff
@@ -110,24 +109,31 @@ async function fetchLocations(map) {
 
 /** Builds a HTML element to display the location's data and a vote button. */
 function buildInfoWindowVote(title, voteCount, note, keyString) {
-  let titleContainer = document.createElement('span');
-  titleContainer.setAttribute('id', 'displayTitle');
-  titleContainer.innerText = title;
-
-  let noteContainer = document.createElement('span');
-  noteContainer.setAttribute('id', 'displayNote');
-  noteContainer.innerText = note;
+  const titleContainer = createSpanContainer(title, 'displayTitle');
+  const noteContainer = createSpanContainer(note, 'displayNote');
+  const voteContainer = createSpanContainer(voteCount, 'displayVoteCount');
 
   const button = document.createElement('button');
+  button.setAttribute('id', 'voteButton');
   button.appendChild(document.createTextNode('VOTE'));
   button.onclick = () => {
     MeetingLocationDAO.updateLocation(keyString);
```

Comment:
Did you need an `async` keyword here?

Reply:
Ahhh, yup. I do. Thanks for pointing that out :)